process.c

```diff
@@ -1155,46 +1167,31 @@ rb_process_status_wait(rb_pid_t pid, int flags)
 
 /*
  *  call-seq:
- *     Process::Status.wait(pid=-1, flags=0)      -> Process::Status
+ *     Process::Status.wait(pid = -1, flags = 0) -> Process::Status
  *
- *  Waits for a child process to exit and returns a Process::Status object
- *  containing information on that process. Which child it waits on
- *  depends on the value of _pid_:
+ *  Like Process.wait, but returns a Process::Status object
+ *  (instead of an integer pid or nil).
  *
- *  > 0::   Waits for the child whose process ID equals _pid_.
+ *  If there are child processes,
+ *  waits for a child process to exit and returns a Process::Status object
+ *  containing information on that process;
+ *  sets thread-local variable <tt>$?</tt>:
  *
- *  0::     Waits for any child whose process group ID equals that of the
- *          calling process.
+ *    Process.spawn('cat /nop') # => 1155880
+ *    Process::Status.wait      # => #<Process::Status: pid 1155880 exit 1>
+ *    $?                        # => #<Process::Status: pid 1155508 exit 1>
  *
- *  -1::    Waits for any child process (the default if no _pid_ is
- *          given).
+ *  If there is no child process,
+ *  returns an "empty" Process::Status object
+ *  that does not represent an actual process;
+ *  does not set thread-local variable <tt>$?</tt>:
  *
- *  < -1::  Waits for any child whose process group ID equals the absolute
- *          value of _pid_.
+ *    Process::Status.wait # => #<Process::Status: pid -1 exit 0>
+ *    $?                   # => #<Process::Status: pid 1155508 exit 1> # Unchanged.
  *
- *  The _flags_ argument may be a logical or of the flag values
- *  Process::WNOHANG (do not block if no child available)
- *  or Process::WUNTRACED (return stopped children that
- *  haven't been reported). Not all flags are available on all
- *  platforms, but a flag value of zero will work on all platforms.
```

Comment:
The new documentation seems not talk about the `flags` argument at all.

Reply:
Meant to refer explicitly to Process.wait.  Done,